src/pages/Thanks/ThanksPage.vue

```diff
@@ -473,6 +487,13 @@ export default {
 			fragment: experimentVersionFragment,
 		}) || {};
 		this.enableMayChallengeHeader = shareChallengeExpData?.version === 'b';
+
+		const landedLoanId = this.cookieStore.get('optin-landed-bp');
```

Comment:
What if the user doesn't add to the basket from the BP? Does this experiment only work if the user goes through the BP?

Reply:
Removed for this PR or future ticketed work?